### PR TITLE
433 patch npm check updates installeren

### PR DESCRIPTION
**Pull request by savitris**
1- Current Issue:
```
> rollup --config rollup.config.ts --configPlugin typescript

[!] Error: Cannot find entry for plugin "typescript". The plugin needs to export a function either as "default" or "typescript" for Rollup to recognize it.
```

2- Current Issue:
```
Failed to compile.

./src/app/[locale]/layout.tsx
Module not found: Can't resolve '@frameless/ui/dist/bundle.css'

https://nextjs.org/docs/messages/module-not-found
```

3- Current Issue:
```
Failed to compile.

./src/components/Markdown/index.tsx
Module not found: Can't resolve '@frameless/ui'

https://nextjs.org/docs/messages/module-not-found
```

4- Current Issue:
```
./src/components/index.tsx
Module not found: Can't resolve '@frameless/ui'

https://nextjs.org/docs/messages/module-not-found

Import trace for requested module:
./src/app/[locale]/error.tsx

./src/app/[locale]/layout.tsx
Module not found: Can't resolve '@frameless/ui/dist/bundle.css'

https://nextjs.org/docs/messages/module-not-found
```

~~Past Issue: Webpack  
Build failed because of webpack errors~~

### CODE REVIEW

File: packages/strapi-tiptap-editor/package.json [hunk 1]
```diff
@@ -60,19 +60,26 @@
...
-    "@utrecht/component-library-css": "3.0.1-alpha.4",
-    "@utrecht/component-library-react": "3.0.1-alpha.4",
-    "@utrecht/design-tokens": "1.0.0-alpha.617",
-    "classnames": "2.3.2",
+    "@utrecht/component-library-css": "3.0.1-alpha.18",
+    "@utrecht/component-library-react": "3.0.1-alpha.18",
+    "@utrecht/design-tokens": "1.0.0-alpha.631",
+    "classnames": "2.3.3",
+    "i": "0.3.7",
+    "npm": "10.4.0",
```
[the remark this answers is not in the document]
Comment (savitris) [reply packages/strapi-tiptap-editor/package.json:68]: Ik deed per ongeluk een `yarn add npm i <node module>` en het is zo snel gegaan er was geen tijd om te cancelen. Ik zag niet dat deze toegevoegd waren

File: apps/kennisbank-dashboard/package.json [hunk 2]
```diff
@@ -17,17 +16,23 @@
...
+    "@strapi/plugin-graphql": "4.14.6",
+    "@strapi/plugin-i18n": "4.14.6",
+    "@strapi/plugin-users-permissions": "4.14.6",
+    "@strapi/strapi": "4.14.6",
     "better-sqlite3": "8.3.0",
     "pg": "8.11.3",
+    "react": "18.0.0",
+    "react-dom": "18.0.0",
+    "react-is": "18.2.0",
+    "react-router-dom": "5.2.0",
```
Comment (AliKdhim87) [inline apps/kennisbank-dashboard/package.json:28]: I think that the dependencies mentioned are not required until the version `@strapi/strapi: 4.14.6`.

File: apps/kennisbank-dashboard/package.json [hunk 3]
```diff
@@ -17,17 +16,23 @@
...
     "slugify": "1.6.6",
     "strapi-plugin-config-sync": "1.2.3",
     "strapi-plugin-publisher": "1.5.7",
     "strapi-plugin-slugify": "2.3.8",
-    "strapi-prometheus": "1.8.1"
+    "strapi-prometheus": "1.8.1",
+    "styled-components": "5.3.3",
```
Comment (AliKdhim87) [inline apps/kennisbank-dashboard/package.json:34]: Same here

File: apps/kennisbank-dashboard/package.json [hunk 4]
```diff
@@ -17,17 +16,23 @@
...
+    "webpack": "5.90.2"
```
Comment (AliKdhim87) [inline apps/kennisbank-dashboard/package.json:35]: Could you clarify the reason for installing this dependency?

File: apps/vth-dashboard/package.json [hunk 5]
```diff
@@ -18,18 +17,24 @@
...
-    "@strapi/strapi": "4.14.4",
+    "@strapi/plugin-graphql": "4.14.6",
+    "@strapi/plugin-i18n": "4.14.6",
+    "@strapi/plugin-users-permissions": "4.14.6",
+    "@strapi/strapi": "4.14.6",
     "better-sqlite3": "8.3.0",
     "pg": "8.11.3",
+    "react": "18.0.0",
+    "react-dom": "18.0.0",
+    "react-is": "18.2.0",
```
Comment (AliKdhim87) [inline apps/vth-dashboard/package.json:28]: Do we need this package?

File: apps/pdc-dashboard/package.json [hunk 6]
```diff
@@ -23,19 +22,25 @@
...
+    "@strapi/plugin-graphql": "4.14.6",
+    "@strapi/plugin-i18n": "4.14.6",
+    "@strapi/plugin-users-permissions": "4.14.6",
+    "@strapi/strapi": "4.14.6",
     "better-sqlite3": "8.3.0",
     "pg": "8.11.3",
+    "react": "18.0.0",
+    "react-dom": "18.0.0",
+    "react-is": "18.2.0",
+    "react-router-dom": "5.2.0",
```
Comment (AliKdhim87) [inline apps/pdc-dashboard/package.json:34]: Are these dependencies necessary?

File: apps/pdc-dashboard/package.json [hunk 7]
```diff
@@ -23,19 +22,25 @@
...
     "slugify": "1.6.6",
     "strapi-plugin-config-sync": "1.2.3",
     "strapi-plugin-import-export-entries": "1.23.1",
-    "strapi-plugin-notes": "1.1.2",
+    "strapi-plugin-notes": "1.1.3",
     "strapi-plugin-publisher": "1.5.7",
     "strapi-plugin-slugify": "2.3.8",
-    "strapi-prometheus": "1.8.1"
+    "strapi-prometheus": "1.8.1",
+    "styled-components": "5.3.3",
```
Comment (AliKdhim87) [inline apps/pdc-dashboard/package.json:42]: Same here

File: apps/pdc-frontend/package.json [hunk 8]
```diff
@@ -22,32 +22,35 @@
...
     "i18next": "22.5.1",
     "i18next-browser-languagedetector": "7.0.2",
     "i18next-resources-to-backend": "1.1.4",
     "is-absolute-url": "4.0.1",
     "next": "13.5.6",
-    "next-sitemap": "4.1.3",
-    "react": "18.2.0",
-    "react-dom": "18.2.0",
+    "next-sitemap": "4.1.8",
+    "prop-types": "15.8.1",
```
Comment (AliKdhim87) [inline apps/pdc-frontend/package.json:41]: Is this dependency required?

File: .yarnrc.yml [hunk 9]
```diff
@@ -0,0 +1,11 @@
+defaultSemverRangePrefix: ""
+
+enableImmutableInstalls: false
+
+nodeLinker: node-modules
+
+yarnPath: .yarn/releases/yarn-4.1.0.cjs
```
Comment (AliKdhim87) [inline .yarnrc.yml:7]: I think these changes happened because of the switch to Yarn version 4

**Pull request closed without merging**